content/blog/2020/07/2020-07-24-custom-distribution-service-phase-2.adoc

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: "Custom Distribution Service : Phase 2 Blogpost"
+tags:
+- service
+- distribution
+- cloud-native
+- gsoc
+- gsoc2020
+- packaging
+- platform-sig
+author: sladyn98
+opengraph:
+  image: /images/gsoc/cds_gsoc.png
+---
+
+
+Hello everyone,
+It is time to wrap up another successfull phase for the link:https://github.com/jenkinsci/custom-distribution-service[cutsom distribution service project], we have incorporated most of the features that we had planned at the start of the phase. It has been an immense learning curve for me and the entire team.
+
+To understand what the project is about and the past progress, please refer to the phase one blogpost
+
+== New Features Added
+
+We targeted three major features for this phase namely the addition of community configurations, war download feature, and the ability to filter plugins. Additional minor features included the searching of community configurations as well as the ability to create pull requests from the front-end of the service.
+
+I have spoken about majority of the features in detail my mid-term blog post link:https://sladyn98.netlify.app/blog/gsoc_midterm/[here]. 
```

Comment:
This blog post is very dependent on the phase 1 blogpost.  Is there a timeframe when that is going to go in?

Reply:
I think I can add more tasks that happened this week